cli/cmd/env/configure.go

```diff
@@ -105,15 +106,33 @@ func ConfigureCmd(cfg *config.Config) *cobra.Command {
 			if err != nil {
 				return fmt.Errorf("failed to update variables %w", err)
 			}
-
 			cmdutil.SuccessPrinter("Updated project variables")
+
+			if !cmd.Flags().Changed("reconcile") {
+				triggerReconcile = cmdutil.ConfirmPrompt("Do you want to reconcile deployment", "", triggerReconcile)
+			}
+
+			if triggerReconcile {
+				project, err := client.GetProject(ctx, &adminv1.GetProjectRequest{OrganizationName: cfg.Org, Name: projectName})
+				if err != nil {
+					return err
+				}
+
+				_, err = client.TriggerReconcile(ctx, &adminv1.TriggerReconcileRequest{DeploymentId: project.ProdDeployment.Id})
+				if err != nil {
+					warn.Printf("Reconcile failed. Trigger reconcile again with `rill project reconcile` if required.")
+					return err
+				}
+				cmdutil.SuccessPrinter("Triggered reconcile successfully.")
+			}
```

Comment:
I'm not sure triggering reconcile makes sense. Reconcile will do nothing if sources are already ingested and models deployed, because the variables are not part of the dependency DAG (we should change that in the next reconcile).  You would need to use `TriggerRefreshSources` to re-ingest sources or use `TriggerRedeploy` to re-build the whole project.

We should either give people the option to chose between a) refresh sources or b) trigger redeploy. Or just ask people if they want to redeploy or not.

Reply:
I was wrongly under the impression that we look at the file content to check if artifact has changed but looks like we are only looking at the file timestamps so yeah in this case `reconcile` will make no sense. I will change to always `redeploy`.